index.js

```diff
@@ -7,8 +7,8 @@ var Parser = function (app, options) {
 
     options = options || {};
 
-    var ext = '.' + app.get('view engine'),
-        partials = traverse(app.get('views'), ext, options.prefix);
+    var ext = options.ext || '.' + app.get('view engine') || 'html',
```

Comment:
The default extension needs a '.' at the start and an associated test. Can we allow `options.ext` to have an optional '.' as the prefix?